integration-tests/src/test/java/tachyon/client/TachyonFileUpdateIntegrationTest.java

```diff
@@ -44,6 +44,9 @@
 
   @Before
   public final void before() throws IOException {
+    // Disable hdfs client caching to avoid file system close() affecting other clients
+    System.setProperty("fs.hdfs.impl.disable.cache", "true");
```

Comment:
Should we put this in `beforeClass` rather than in `before`, and also the `clearProperty`?